code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -99,6 +99,55 @@ datum
 						M.HealDamage("All", clamp(1 * volume, 0, 10), clamp(1 * volume, 0, 10)) //put a cap on instant healing
 						if(prob(15))
 							boutput(M, "<span class='notice'>The milk comforts your [pick("boanes","bones","bonez","boens","bowns","beaunes","brones","bonse")]!</span>")
+		fooddrink/milk_powder
+			// It doesn't help with spice, unlike liquid milk.
+			// This is partly because I feel like it'd work that way, and partly because it's a hassle.
+			name = "milk powder"
+			id = "milk_powder"
+			description = "Powdered milk, made by evaporating milk. Add to water to make it into milk again."
+			reagent_state = SOLID
+			fluid_r = 255
+			fluid_g = 255
+			fluid_b = 255
+			transparency = 255
+			taste = "overly milky"
+			thirst_value = 0.6
+			bladder_value = -0.2
+			viscosity = 0.3
+
+		fooddrink/super_milk
```

Comment:
why isn't this just a child of milk

Reply:
mainly because i forgot i could do that until you said it